ferrocene/doc/safety-manual/src/rustfmt/constraints.rst

```diff
@@ -6,30 +6,22 @@
 Constraints
 ===========
 
-Install version control system
-------------------------------
-
-.. id:: RUSTFMT_CSTR_0010_HAVE_VCS
-
-Associated requirement ID: :id:`RUSTFMT_AVD_CHECK_DIFF_001`
-
-Before executing ``rustfmt``, the user shall ensure that the a version control
-system (VCS, e.g. ``git``) is installed and used.
-
 Review diff
 -----------
 
-.. id:: RUSTFMT_CSTR_0020_USE_VCS
+.. id:: RUSTFMT_CSTR_0010_CHECK_DIFF
 
 Associated requirement ID: :id:`RUSTFMT_AVD_CHECK_DIFF_001`
 
-After executing ``rustfmt``, the user shall review, with the help of the VCS, the
-modifications ``rustfmt`` performed and ensure that there are no semantic changes.
+After executing ``rustfmt``, the user shall review the modifications
+``rustfmt`` performed and ensure that there are no semantic changes.
+
+Using a version control system for that makes sense, but is not mandated.
```

Comment:
would not mention version control at all... users can decide what *makes sense* for them